### ISSUE TITLE
Missing line breaks

### ISSUE THREAD

**Issue by artasbartas** [contentful-labs/lookbook-example#2]
Noticed that the rendered page ignores line breaks in text. See snapshots.

Editor view
<img width="584" alt="screen shot 2015-09-28 at 13 42 57" src="https://cloud.githubusercontent.com/assets/794132/10134543/f87841ba-65e6-11e5-8ad6-a69733c6f158.png">

Frontend view
<img width="713" alt="screen shot 2015-09-28 at 13 43 09" src="https://cloud.githubusercontent.com/assets/794132/10134552/061e6ee8-65e7-11e5-8e3e-99735f87b391.png">

**Comment by geigerzaehler** [comment]
Yep, in markdown a single line break does not have any significance. Double line break would start a new paragraph with a spacing too wide. I would recommend separating the items with periods.

**Comment by timdp** [comment]
~~You can insert a line break by ending your line with two spaces though. This is a discrepancy in Contentful's Markdown editor.~~

Contentful uses these options for [marked](https://www.npmjs.com/package/marked):

``` js
{
  "gfm": true,
  "tables": true,
  "breaks": true,
  "smartypants": true,
  "sanitize": false
}
```

Source: copied from Contentful source code in Chrome DevTools.

**Comment by sidonaldson** [comment]
I seem to be going crazy here. I still can't get any line-breaks to show - at all!
Even in the Contentful preview UI it refuses to show any. Is it meant to be double space and return? Or double return?

**Comment by timdp** [comment]
@sidonaldson The last time I checked, two spaces followed by a return worked.

**Comment by sidonaldson** [comment]
thanks @timdp does that work for multiple line-breaks?

**Comment by timdp** [comment]
@sidonaldson I'm not sure if Markdown has a concept of multiple line breaks outside of preformatted text. I think anything over one line break is basically a new paragraph. You can always run marked yourself thanks to Tonic; that should give you the same behavior as Contentful. [Here's an example](https://tonicdev.com/56cc68597512050c002bdda1/573c5727691f641100ad6f84).

**Comment by sidonaldson** [comment]
@timdp you're right. I've been using a custom renderer which includes a spacer. I now realise Markdown is performing as expected and essentially creating clean html. Thanks

**Comment by bythelabo** [comment]
Hey 

I have the same issue using Contentful Rich Text 
![Screen Shot 2019-04-17 at 5 27 30 PM](https://user-images.githubusercontent.com/49015375/56277047-2cd81e00-6136-11e9-84c5-9cb783550678.jpg)

the rendering ignores the line break. 
Expected result: have line break and not another paragraph <p>

What is the solution? 

thank you!

**Comment by zeekrey** [comment]
This works for me: https://github.com/contentful/rich-text/issues/96#issuecomment-511100434
